setup.py

```diff
@@ -2,7 +2,4 @@
 
 from setuptools import setup
 
-setup(
-    setup_requires=['pbr'],
-    pbr=True,
-)
+setup()
```

Comment:
Technically, this file could probably be deleted, but is just left minimal for backwards compat